Makefile

```
@@ -1251,13 +1270,32 @@ clear-os-data: ## Clear OpenSearch data volume
 # FLOW MANAGEMENT
 ######################
 
-flow-upload: ## Upload flow to Langflow
-	@echo "$(YELLOW)Uploading flow to Langflow...$(NC)"
+flow-upload: ## Create or update a flow in Langflow from a JSON file (id taken from the file itself)
```

checkmake (0.3.2)

[warning] Target body for "flow-upload" exceeds allowed length of 5 lines (25).

(maxbodylength)